style2.css

```diff
@@ -1,261 +1,261 @@
+    height: fit-content;
+    margin-left: 15px;
+  }
+
+  .middle-image {
+    display: none;
+  }
+
+  .middle-section {
+    flex-direction: column;
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    flex-wrap: wrap;
+    display: flex;
+    /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
+    flex-direction: row;
+    /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
+    flex-wrap: wrap;
+    align-content: center;
+    margin-left: 141.5px;
+    margin-right: 118px;
+    margin-bottom: 130px;
+  }
+
+  .middle-image1-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 20px;
+    margin-left: 26px;
+  }
+
+  .middle-1-container h1 {
+    margin-top: 0;
+  }
+
+  .middle-1-container p {
+    margin-left: 14px;
+  }
+
+  .about-container {
+    background-image: url("lowerimage.png");
+    background-repeat: no-repeat;
+    background-position: center right;
+    background-position-y: 100px;
+    background-size: 672px 352px;
+  }
+
+  .about h1 {
+    font-family: "Crete Round", serif;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+    padding-top: 120px;
+    margin-left: 142px;
+    margin-bottom: 24px;
+  }
+
+  .about p {
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    margin-left: 142px;
+    width: 566px;
+    text-align: left;
+  }
+
+  .about button {
+    height: 48px;
+    background: #f55800;
+    font-family: Inter, sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    margin-left: 142px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+  }
+
+  #about-line {
+    margin-left: 142px;
+    margin-right: 142px;
+    border: 1px solid #fff;
+    margin-top: 209px;
+  }
+
+  .daimond {
+    width: 18px;
+    height: 18px;
+    display: inline-flex;
+    background: #ff6b00;
+    transform: rotate(-45deg);
+    margin-block-end: auto;
+  }
+
+  .rectangle {
+    width: 18px;
+    height: 18px;
+    left: 12px;
+    top: 11px;
+    background: #ff6b00;
+    margin-block-end: auto;
+    margin-right: 12px;
+    margin-top: 12px;
+  }
+
+  .circle {
+    width: 18px;
+    height: 18px;
+    left: 12px;
+    top: 11px;
+    background: #ff6b00;
+    border-radius: 150px;
+    margin-block-end: auto;
+    margin-right: 12px;
+  }
+
+  .about-discription-container {
+    display: flex;
+    margin-top: 146px;
+  }
+
+  .shapes-daimond {
+    margin-top: -90px;
+    margin-left: 20px;
+  }
+
+  .shapes-rectangle {
+    margin-left: 120px;
+  }
+
+  .shapes-circle {
+    margin-left: 120px;
+  }
+
+  .about-container h3 {
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    display: flex;

```

Comment:
- [ ] Please fix the alignment of this section to be in the middle with making attention to margins and widths and the positioning of the shapes to be beside the heading and try to stick as much as possible to the given Figma design.
and fix it in the mobile version as well.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160948851-b2ef9d56-9789-42d1-8c4d-a5b1e4064755.png)  | ![image](https://user-images.githubusercontent.com/68796232/160948866-9421b708-9e00-440d-abcb-af3765a47911.png)  |